src/Momento.Sdk/Auth/EnvironmentTokenProvider.cs

```diff
@@ -0,0 +1,38 @@
+namespace Momento.Sdk.Auth;
+
+using System;
+using Momento.Sdk.Exceptions;
+using Momento.Sdk.Internal;
+
+public class EnvironmentTokenProvider : ICredentialProvider
+{
+
+    private string _authToken;
+    private string _controlEndpoint;
+    private string _cacheEndpoint;
```

Comment:
why not remove these and include a private set on the implementations below? You don't have to put the private set in the interface to put it on the implementation

Reply:
Thanks, fixed in fb214b5.